dvc/commands/remote.py

```diff
@@ -113,8 +113,8 @@ def run(self):
 class CmdRemoteList(CmdRemote):
     def run(self):
         conf = self.config.read(self.args.level)
-        for name, conf in conf["remote"].items():
-            ui.write(name, conf["url"], sep="\t")
+        for name, conf_val in conf["remote"].items():
+            ui.write(name, conf_val["url"], sep="\t")
```

Comment:
Sorry also unrelated... [Tests](https://github.com/iterative/dvc/runs/5765753773?check_suite_focus=true) were complaining about this (lint check), `for loop that reassigns the iterable it is iterating with each iterable value`.